legate/core/corelib.py

```diff
@@ -1,4 +1,3 @@
-# Copyright 2021-2022 NVIDIA Corporation
 #
```

Comment:
I assume we still want a copyright header?

Reply:
Oh weird, not sure why/how that happened

Reply:
fixed (two commits, tried to be to clever with GH web ui suggestion)